pycaption/scc/__init__.py

```diff
@@ -474,7 +503,7 @@ class SCCWriter(BaseWriter):
     def __init__(self, *args, **kw):
         super().__init__(*args, **kw)
 
-    def write(self, caption_set):
+    def write(self, caption_set, src_fps=30.0):
```

Comment:
I don't think the name src_fps is a good fit, as this is no longer the fps dedicated to the source file, but to the output file. Maybe calling it simply fps would be easier?